review-notifications/src/Github.js

```diff
@@ -92,16 +103,51 @@ class Github extends Component {
             mentionedPR: result.mentionedPR ? result.mentionedPR : [],
             reviewedPR: result.reviewedPR ? result.reviewedPR : [],
             hasData: true,
+            fetchingFollowedRepos: result.fetchingFollowedRepos
```

Comment:
if I'm not mistaken this can be simply `fetchingFollowedRepos: result.fetchingFollowedRepos`